### PR TITLE
change the base styles of dropdown and display menu

### PR DESCRIPTION
**Pull request by shravanthivr**
Update the Dropdown styles with scrollable content

Before:
Screenshot 2024-10-22 at 9 11 55 PM

After:
Screenshot 2024-10-28 at 9 05 14 PM
Screenshot 2024-10-28 at 9 05 25 PM

### CODE REVIEW

**Comment by shravanthivr** [comment]
[Screen Recording 2024-10-29 at 4.40.57 PM.zip](https://github.com/user-attachments/files/17562931/Screen.Recording.2024-10-29.at.4.40.57.PM.zip)
This is the issue I am seeing, I still need to fix the input text border style

**Comment by dallasbrittany** [comment]
> Screen Recording 2024-10-29 at 4.40.57 PM.zip

Thanks for sharing this. Yeah, we're going to have to look into it. It's not related to this PR though, right? It existed as a problem before?

File: src/components/signup/OrgLookup.tsx [hunk 1]
```diff
@@ -129,19 +127,15 @@ const OrgLookup = ({
...
-   * set to "false" when an organization is selected
-   */
-  const [disableSubmit, setdisableSubmit] = useState<boolean>(true);
+  //Form disabled or active flag, set to "false" when an organization is selected
+  const [disableSubmit, setDisableSubmit] = useState<boolean>(true);
   const [askConfirmation, setAskConfirmation] = useState<boolean>(false);
+  const [isOpen, setIsOpen] = useState(false);
+  const [searchTerm, setSearchTerm] = useState('');
 
+  //List of avialable organizations
```
Comment (dallasbrittany) [inline src/components/signup/OrgLookup.tsx:137]: It's minor but do you mind adding a space before List and before Form after the `//`?

**Comment by shravanthivr** [comment]
> > Screen Recording 2024-10-29 at 4.40.57 PM.zip
> 
> Thanks for sharing this. Yeah, we're going to have to look into it. It's not related to this PR though, right? It existed as a problem before?

Yes, it is a problem fetching the details from API

**Comment by shravanthivr** [comment]
Need review for this PR too, somehow got closed

File: src/components/signup/OrgLookup.tsx (hunk 1 above)
Comment (shravanthivr) [reply src/components/signup/OrgLookup.tsx]: Need review for this PR too, not sure how it got closed

**Review by dallasbrittany — APPROVED** [review]

**Pull request merged**